src/components/Capture/index.js

```diff
@@ -103,23 +103,31 @@ class Capture extends Component {
     this.validateCapture(message.id, valid)
   }
 
-  handleImage = (base64ImageLossy, base64capture, file) => {
-    if (!base64capture) {
+  handleBase64 = (lossyBase64, base64) => {
+    base64toFile(base64,
+      file => this.handleImage(lossyBase64, file)
```

Comment:
I still find this hard to follow. Can you give the callback function a name?

e.g. `const fileCallback = file => this.handleImage(lossyBase64, file)`